src/NHibernate/Event/EventListeners.cs

```diff
@@ -99,12 +101,12 @@ static EventListeners()
 		private IPreLoadEventListener[] preLoadEventListeners = new IPreLoadEventListener[]
 		                                                        	{new DefaultPreLoadEventListener()};
 
-		private IPreDeleteEventListener[] preDeleteEventListeners = new IPreDeleteEventListener[] {};
-		private IPreUpdateEventListener[] preUpdateEventListeners = new IPreUpdateEventListener[] {};
-		private IPreInsertEventListener[] preInsertEventListeners = new IPreInsertEventListener[] {};
-		private IPostDeleteEventListener[] postDeleteEventListeners = new IPostDeleteEventListener[] {};
-		private IPostUpdateEventListener[] postUpdateEventListeners = new IPostUpdateEventListener[] {};
-		private IPostInsertEventListener[] postInsertEventListeners = new IPostInsertEventListener[] {};
+        private IPreDeleteEventListener[] preDeleteEventListeners = new IPreDeleteEventListener[] { defaultEventEntityListener };
+        private IPreUpdateEventListener[] preUpdateEventListeners = new IPreUpdateEventListener[] { defaultEventEntityListener };
+        private IPreInsertEventListener[] preInsertEventListeners = new IPreInsertEventListener[] { defaultEventEntityListener };
+        private IPostDeleteEventListener[] postDeleteEventListeners = new IPostDeleteEventListener[] { defaultEventEntityListener };
+        private IPostUpdateEventListener[] postUpdateEventListeners = new IPostUpdateEventListener[] { defaultEventEntityListener };
+        private IPostInsertEventListener[] postInsertEventListeners = new IPostInsertEventListener[] { defaultEventEntityListener };

```

Comment:
Sorry, the code base uses tabs. Identation should be fixed here, and new .cs files should re-idented with tabs.